examples/RNN_toy_example.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
```

Comment:
Can you please write here a text description of the learning task solved by the script, just like you explained it in the PR description?